crates/fluvio-spu-schema/src/server/fetch_offset.rs

```diff
@@ -113,6 +113,10 @@ pub struct FetchOffsetPartitionResponse {
 
     /// Last readable offset
     pub last_stable_offset: i64,
+
+    /// Consumer offset
```

Comment:
can you add more description on what is requested?

Reply:
Shouldn't this be list of consumer id (string)?

Reply:
I changed to `Last committed consumer offset`, what do you think?